enterprise/frontend/src/metabase-enterprise/metabot/components/MetabotAdmin/MetabotAdminPage.tsx

```diff
@@ -214,6 +214,8 @@ function MetabotConfigurationPane({
       {isOpen && (
         <CollectionPickerModal
           title={t`Select a collection`}
+          shouldDisableItem={(item) => item.id === "root"}
+          canSelectItem={(item) => item && item.id !== "root"}
```

Comment:
```suggestion
          shouldDisableItem={isRootCollection}
          canSelectItem={(item) => item && !isRootCollection(item)}
```

Reply:
don't turn it into a Big Thing. just hardcoding root everywhere, and not handling `null` bites us again and again